tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2190,15 +2190,16 @@ def test_histogram_function(self):
 
         assert response.status_code == 200, response.content
         data = response.data["data"]
-        assert len(data) == 10
+        assert len(data) == 11
```

Comment:
If the request is for 10 results why do 11 come back?

Reply:
There is an edge case where if the max duration is exactly divisible by the number of buckets, then there will be an extra row with just the results with that exact duration. It comes up in the tests because I engineered all the results to be divisible so it was easier to understand what was happening.